src/libtriton/bindings/python/objects/pyImmediate.cpp

```diff
@@ -111,7 +111,7 @@ namespace triton {
 
       static PyObject* Immediate_getBitvector(PyObject* self, PyObject* noarg) {
         try {
-          return PyBitvector(*PyImmediate_AsImmediate(self));
+          return PyBitsVector(*PyImmediate_AsImmediate(self));
```

Comment:
As we only user getter, it is not really important but if we succeed to use inheritance with the CPython API, we could use it and keep the initial Immediate Object.